app/views/facilities/_members-table-row.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr class="bg-white border-b border-gray-200" x-description="Odd row">
+  <td class="px-6 py-4 whitespace-nowrap text-sm font-medium text-gray-900">
+    <%= nurse.full_name %>
+  </td>
+  <td class="px-6 py-4 whitespace-nowrap text-sm font-medium text-gray-900">
+    <%= nurse.phone %>
+  </td>
+  <td class="px-6 py-4 whitespace-nowrap text-center text-sm font-medium">
+    <% if policy(@facility).unassign_facility? %>
+      <%= form_with scope: :facility, url: unassign_facility_path, method: :put do |form| %>
+        <%= form.hidden_field "facility_id", value: @facility.id %>
+        <%= form.hidden_field "nurse_id", value:nurse.id %>
+        <%= button_tag "", {
+                      class: "bg-red-50 rounded-md p-1.5 text-red-500 hover:bg-red-100 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-red-50 focus:ring-red-600",
+                      onClick: "this.form.submit()"
+                      } do %>
```

Comment:
I hope we can use `form.submit` which automatically submits on click. 🤔

Reply:
See if `form.submit` supports enclosing svg like `button_tag`

Reply:
Like this 👇👇

https://github.com/coronasafe/arike/blob/2f7f6334c3b4c64298ba4bf3c9c375df336b436a/app/views/devise/sessions/new.html.erb#L54

🔥

Reply:
Like this 👇👇

https://github.com/coronasafe/arike/blob/main/app/views/devise/sessions/new.html.erb#L54:L59

🔥

Reply:
@ProCode2